config.toml

```diff
@@ -1,5 +1,6 @@
-baseurl = "https://jakarta.ee/"
-DefaultContentLanguage = "en"
+#baseurl = "https://jakarta.ee/"
```

Comment:
these changes should be reverted, as they are important to the production version of the site